Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/approximate_convex_decomposition.h

```diff
@@ -0,0 +1,1639 @@
+ *
+ * \tparam OutputIterator must be an output iterator accepting variables of type `std::pair<std::vector<geom_traits::Point_3>, std::vector<std::array<unsigned int, 3> > >`.
+ *
+ * \tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+ *
+ * \param mesh the input mesh to approximate by convex hulls
+ * \param out_hulls output iterator into which convex hulls are recorded
+ * \param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+ *
+ * \cgalNamedParamsBegin
+ *
+ *   \cgalParamNBegin{maximum_number_of_voxels}
+ *     \cgalParamDescription{gives an upper bound of the number of voxels. The longest bounding box side will have a length of the cubic root of `maximum_number_of_voxels` rounded down. Cannot be smaller than 64. }
+ *     \cgalParamType{unsigned int}
+ *     \cgalParamDefault{1,000,000}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{maximum_depth}
+ *     \cgalParamDescription{maximum depth of hierarchical splits}
+ *     \cgalParamType{unsigned int}
+ *     \cgalParamDefault{10}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{maximum_number_of_convex_hulls}
+ *     \cgalParamDescription{maximum number of convex hulls produced by the method}
+ *     \cgalParamType{unsigned int}
+ *     \cgalParamDefault{16}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{volume_error}
+ *     \cgalParamDescription{maximum difference in fraction of volume of the local convex hull with the sum of voxel volumes. If surpassed, the convex hull will be split if the `maximum_depth` has not been reached yet.}
+ *     \cgalParamType{double}
+ *     \cgalParamDefault{0.01}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{split_at_concavity}
+ *     \cgalParamDescription{split the local box of the convex hull at the concavity along the longest axis of the bounding box. Otherwise, split in the middle of the longest axis (faster, but less precise).}
+ *     \cgalParamType{Boolean}
+ *     \cgalParamDefault{true}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{vertex_point_map}
+ *     \cgalParamDescription{a property map associating points to the vertices of `mesh`}
+ *     \cgalParamType{a class model of `ReadablePropertyMap` with `boost::graph_traits<FaceGraph>::%vertex_descriptor`
+ *                    as key type and `%Point_3` as value type}
+ *     \cgalParamDefault{`boost::get(CGAL::vertex_point, mesh)`}
+ *     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t`
+ *                     must be available in `FaceGraph`.}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{concurrency_tag}
+ *     \cgalParamDescription{a tag indicating if the task should be done using one or several threads.}
+ *     \cgalParamType{Either `CGAL::Sequential_tag`, or `CGAL::Parallel_tag`, or `CGAL::Parallel_if_available_tag`}
+ *     \cgalParamDefault{`CGAL::Parallel_if_available_tag`}
```

Comment:
So far, `concurrency_tag`, whether it was passed as a template parameter or NP, has always defaulted to `Sequential_tag` in the header (and `Parallel_if_available_tag` can be in the example), with IIRC the reasoning being that it is simpler, for example to control exactly at which level you want to parallelize a stack of potentially parallel functions.